sdks/python/apache_beam/ml/transforms/handlers.py

```diff
@@ -83,20 +85,52 @@
 tft_process_handler_output_type = typing.Union[beam.Row, Dict[str, np.ndarray]]
 
 
+class DataCoder:
+  def __init__(self, exclude_columns=None):
+    """
+    Uses PickleCoder to encode/decode the dictonaries.
+    Args:
+      exclude_columns: list of columns to exclude from the encoding.
+    """
+    self.coder = coders.registry.get_coder(Any)
+    self.exclude_columns = exclude_columns
+
+  def set_unused_columns(self, exclude_columns):
+    self.exclude_columns = exclude_columns
+
+  def encode(self, element):
+    if not self.exclude_columns:
```

Comment:
interesting. Is it possible for `exclude_columns` be emtpy? I'd imagine it could rather be the opposite, where all columns are being processed, so there is nothing to encode/decode.